packages/lambda/src/oauth/login/google.ts

```diff
@@ -35,26 +34,23 @@ interface GoogleUser {
 export class GoogleLoginHandler extends OAuthLoginHandler {
   oAuthProvider = OAuthProvider.Google;
 
+  /**
+   * https://developers.google.com/identity/protocols/oauth2/web-server#exchange-authorization-code
+   */
   async fetchGoogleOAuthTokens(code: string): Promise<GoogleOAuthResponse> {
     const baseUrl = "https://oauth2.googleapis.com/token";
 
     const { client_id, client_secret } = await fetchOAuthClientInformation(this.oAuthProvider);
 
-    const queryStrings = new URLSearchParams({
-      code,
-      client_id,
-      client_secret,
-      redirect_uri: `${siteUrl}/api/oauth/google`,
-      grant_type: "authorization_code",
-    });
-
-    const response = await fetch(`${baseUrl}?${queryStrings.toString()}`, {
+    const response = await fetch(baseUrl, {
       method: "POST",
-      // Content-Length wasn't needed initially, but after a few months we suddenly needed it to
-      // avoid a 411 error... https://stackoverflow.com/a/18352423
-      // We should probably start using the official Google APIs npm package once LLRT supports
-      // the necessary Node modules.
-      headers: { "Content-Type": "application/x-www-form-urlencoded", "Content-Length": "0" },
```

Comment:
Makes sense now, we should be using the body instead of query strings. We don't need to provide `Content-Length: 0` when we actually have stuff in the body. No idea why:
- query strings worked initially with no `Content-Length` header
- we suddenly needed the `Content-Length` header after months of it working